src/main/java/org/folio/consortia/domain/entity/UserTenantEntity.java

```diff
@@ -31,6 +31,7 @@ public class UserTenantEntity extends AuditableEntity {
   private TenantEntity tenant;
 
   private Boolean isPrimary;
+  private String centralTenantId;
```

Comment:
This field is not needed in user-tenant entity

Reply:
deleted